frontend/test/metabase/scenarios/admin/datamodel/reproductions/17768-entity-key-showing-binning-options.cy.spec.js

```diff
@@ -0,0 +1,45 @@
+import { restore, openReviewsTable, popover } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { REVIEWS } = SAMPLE_DATASET;
+
+describe.skip("issue 17768", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.request("PUT", `/api/field/${REVIEWS.ID}`, {
+      semantic_type: "type/Category",
+      has_field_values: "list",
+    });
+
+    // Sync "Sample Dataset" schema
+    cy.request("POST", `/api/database/1/sync_schema`);
+    /**
+     * This is a bit fragile and may result in the false positive result, depending on the CI machine that runs tests.
+     * However, 3s should be a plenty of time for the sync to finish.
+     * Although the arbitrary waiting is considered a bad practice, we don't have any other way to determine when the sync is finished.
+     */
+    cy.wait(3000);
```

Comment:
This is likely safe.

But you could hide/unhide the table via `PUT /api/table`, which should be faster, since it's only one table.

Or alternatively, `GET /api/field/${REVIEWS.ID}` and look if `fingerprint` is not null - otherwise then do a short wait and check again (but maximum of 10s or something like that to prevent loop).

Reply:
Wrote a function that checks for `fingerprint`'s value every 100ms. Thanks for the tip.